packages/block/dom.ts

```diff
@@ -113,38 +107,62 @@ export const insertText = (
 ): Text => {
   const node = document.createTextNode(value);
   const childNodes = childNodes$.call(el);
+<<<<<<< HEAD
+  insertBefore$.call(el, node, childNodes[index]);
+=======
```

Comment:
```suggestion
<<<<<<< HEAD
  insertBefore$.call(el, node, childNodes[index]);
=======
```

Wait what?

Reply:
I think that might be part of an older commit, I can't seem to find it in the Files Changed tab